ansible/library/port_alias.py

```diff
@@ -94,6 +94,11 @@ def get_portmap(self, asic_id=None):
         portmap = {}
         aliasmap = {}
         portspeed = {}
+        # Front end interface asic names
+        asicifnames = []
+        # All asic names
+        asicnames = []
```

Comment:
rename to asic_naames ?

Reply:
Renamed to asic_if_names. This will be a map of ASIC: [ list of all interface of that asic ]